.changeset/tall-otters-walk.md

```diff
@@ -0,0 +1,6 @@
+---
+"graphql-manual": patch
```

Comment:
We never need to bump this - it actually breaks changesets a bit. Just a heads up!

```suggestion
```